docs/security.rst

```diff
@@ -310,3 +310,20 @@ standard port 443, you'll need to configure that too. Be aware that super user p
     # Optionally, set the server to listen on the standard SSL port.
     web_server_port = 443
     base_url = http://<hostname or IP>:443
+
+Impersonation
+'''''''''''''
+
+Airflow has the ability to impersonate a unix user while running task
+instances based on the task's ``run_as_user`` parameter, which takes a user's name.
+
+*NOTE* For impersonations to work, Airflow must be run with `sudo` as subtasks are run
```

Comment:
Let me know when you are available.

The section I was referring to was:
```
+            # Give ownership of file to user; only they can read and write
 +            subprocess.call(
 +                ['sudo', 'chown', self.run_as_user, cfg_path]
 +            )
```

sudo chown will only work as the root user.